tests/integration/article.test.js

```diff
@@ -231,7 +231,7 @@ describe("# Articles endpoints", () => {
     });
   });
 });
-describe("Share Articles endpoints", () => {
+describe.skip("Share Articles endpoints", () => {
```

Comment:
This part can be removed, because the tests are being skipped, and we no longer need them in the back0-end, they will be implemented in the front-end.